docs/README_DD_api.md

```diff
@@ -0,0 +1,475 @@
+      - ansible-test-image2
+    ensure: absent
+    unique_names: false
+    operate_on_multiple: true
+
+```
+
+
+
+---
+
+
+## dimensiondata_disk
+Add/Remove/Modify Disks
+
+  * Synopsis
+  * Options
+  * Examples
+
+#### Synopsis
+ Add/Remove/Modify disks for a host in the Dimension Data Cloud.
+
+#### Options
+
+| Parameter     | required    | default  | choices    | comments |
+| ------------- |-------------| ---------|----------- |--------- |
+| wait_poll_interval  |   no  |  2  | |  The amount to time inbetween polling for task completion  |
+| node_ids  |   no  |  | |  A list of server ids to work on.  |
+| region  |   |  na aka North America  | <ul> <li>Regions are defined in Apache libcloud project file = libcloud/common/dimensiondata.py</li>  <li>See https://libcloud.readthedocs.io/en/latest/compute/drivers/dimensiondata.html</li>  <li>{u'Note': u'Values avail in array dd_regions().'}</li> </ul> |  The target region.  |
+| scsi_id  |   yes  |  | |  The scsi_id for the disk.  Which slot the disk should be in.  |
+| state  |   no  |  present  | <ul> <li>present</li>  <li>absent</li> </ul> |  The state you want the hosts to be in.  |
+| wait_timeout  |   no  |  300  | |  Only applicable if wait is true.  This is the amount of time in seconds to wait  |
+| wait  |   no  |  False  | |  Should we wait for the task to complete before moving onto the next.  |
+| speed  |   |  STANDARD  | <ul> <li>STANDARD</li>  <li>ECONOMY</li>  <li>HIGHPERFORMANCE</li> </ul> |  The speed of the disk.  |
+| verify_ssl_cert  |   no  |  True  | |  Check that SSL certificate is valid.  |
+| size  |   no  |  | |  The size of the disk in GB  |
+
+
+ 
+#### Examples
+
+```
+# Note: These examples don't include authorization.
+# You can set these by exporting DIDATA_USER and DIDATA_PASSWORD vars:
+# export DIDATA_USER=<username>
+# export DIDATA_PASSWORD=<password>
+
+# Add a disk to slot 3 that is 40GB and ECONOMY speed
+
+- dimensiondata_disk:
+    node_ids:
+      - '7ee719e9-7ae9-480b-9f16-c6b5de03463c'
+    state: present
+    scsi_id: 3
+    size: 40
+    speed: ECONOMY
+
+# Modify a disk in slot 3 and wait for it to finish before returning success
+
+- dimensiondata_disk:
+    node_ids:
+      - '7ee719e9-7ae9-480b-9f16-c6b5de03463c'
+    state: present
+    scsi_id: 3
+    size: 45
+    speed: HIGHSPEED
+
+# Remove a disk in slot 3
+- dimensiondata_disk:
+    node_ids:
+      - '7ee719e9-7ae9-480b-9f16-c6b5de03463c'
+    state: absent
+    scsi_id: 3
+
+```
+
+
+
+---
+
+
+## dimensiondata_get_unallocated_public_ips
+Get specified number of free addresses; provision to reach requested number.
+
+  * Synopsis
+  * Options
+  * Examples
+
+#### Synopsis
+ G
+ e
+ t
+  
+ s
+ p
+ e
+ c
+ i
+ f
+ i
+ e
+ d
+  
+ n
+ u
+ m
+ b
+ e
+ r
+  
+ o
+ f
+  
+ f
+ r
+ e
+ e
+  
+ a
+ d
+ d
+ r
+ e
+ s
+ s
+ e
+ s
+ ;
+  
+ p
+ r
+ o
+ v
+ i
+ s
+ i
+ o
+ n
+  
+ t
+ o
+  
+ r
+ e
+ a
+ c
+ h
+  
+ r
+ e
+ q
+ u
+ e
+ s
+ t
+ e
+ d
+  
+ n
+ u
+ m
+ b
+ e
+ r
+ .
```

Comment:
Bizarre.


On Tue, Jul 12, 2016 at 11:16 AM, Aimon Bustardo <notifications@github.com>
wrote:

> In docs/README_DD_api.md
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1#discussion_r70491947>
> :
>
> > + e
> > + q
> > + u
> > + e
> > + s
> > + t
> > + e
> > + d
> > +
> > + n
> > + u
> > + m
> > + b
> > + e
> > + r
> > + .
>
> LOL whats up with the one car per line?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1/files/8df60955ffdbeda67007f8a82b7dcc4e23ee7d17#r70491947>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI6_IZyjhsoytEcv218E93jbkws19W1Yks5qU9oFgaJpZM4JKqRJ>
> .
>